public_examples/kvtokencheck/checkKVTokens.py

```diff
@@ -0,0 +1,163 @@
+        logger.critical ("Error Connecting:",errc)
+        return_status = 500
+        return_message = "Failure"
+        return (return_status,return_message,errc)
+    except requests.exceptions.Timeout as errt:
+        logger.critical ("Timeout Error:",errt)
+        return_status = 500
+        return_message = "Failure"
+        return (return_status,return_message,errt)
+    except requests.exceptions.RequestException as e:
+        ## catastrophic error. bail.
+        logger.critical("Your Request had an error: ",  e)
+        logger.critical ("Check parameters")
+        logger.critical("Your Request had an error with Status %s" %(response.status_code))
+        return_status = 500
+        return_message = "Failure"
+        return (response.status_code,return_message,e)
+
+
+def getKVTokens():
+    """ Function to get the tokens """
+    host = os.environ['AKAMAI_API_HOST']
+    baseurl = 'https://%s' % host
+    headers =  {}
+    requestURL = '/edgekv/v1/tokens'
+    payload = {'includeExpired':'true' }
+    status = submit_request(urljoin(baseurl,requestURL),payload,"GET",headers)
+    logger.debug (status[0])
+    if status[0] != 200:
+        logger.info("Request to fetch token info failed, please review below errors")
+        logger.error(status[0])
+        logger.error(status[1])
+        exit(1)
+    else:
+        jsonList = json.loads(status[1].text)
+        return(jsonList)
+
+
+def days_between(date1, date2):
+    """ function to get difference between two dates """
+    date1 = datetime.datetime.strptime(date1, "%Y-%m-%d")
+    date2 = datetime.datetime.strptime(date2, "%Y-%m-%d")
+    return abs((date2 - date1).days)
+
+### Start Processing
+slack_webhook = os.environ['SLACK_WEB_HOOK']
+client_secret = os.environ['AKAMAI_CLIENT_SECRET']
+access_token = os.environ['AKAMAI_ACCESS_TOKEN']
+client_token = os.environ['AKAMAI_CLIENT_TOKEN']
+if isinstance(os.environ['LEAD_TIME'], str):
+    duration = int(os.environ['LEAD_TIME'])
+else:
+    duration = os.environ['LEAD_TIME']
+
+if 'DEBUG' in os.environ:
+    debug_level = True
+else:
+    debug_level = False
+
+## Initialize logger
+logger = logging.getLogger('getAkamaiLocationInfo')
+logger.setLevel(logging.DEBUG)
+# create console handler with a higher log level
+ch = logging.StreamHandler()
+## Create the Format
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+ch.setFormatter(formatter)
+
+if debug_level:
+    ## Create log file handle
+    log_file = os.path.basename(__file__).split(".")[0] + ".log"
+    fh = logging.FileHandler(log_file,'w+')
+    fh.setFormatter(formatter)
+    fh.setLevel(logging.DEBUG)
+    ch.setLevel(logging.INFO)
+    def httpclient_log(*args):
+        """ function to get http debug """
+        logger.debug(" ".join(args))
+        http.client.print = httpclient_log
+        http.client.HTTPConnection.debuglevel = 1
+        logger.addHandler(fh)
```

Comment:
I think there is an indentation issue here, I was not getting any logs in the logfile till I removed the extra indentation.

Reply:
Fixed